examples/dynamic-openapi-joi.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
+
+const fastify = require('fastify')()
+const joiToJson = require('joi-to-json')
```

Comment:
```suggestion
// joi-to-json is one of the packages that can convert joi to json schema
// you are feel free to use other package that have the similar function
const convert = require('joi-to-json')
```

add a comment about `joi-to-json` can be changed to other packages that have similar ability.